tests/ballerina-compiler-api-test/src/test/resources/test-src/symbols/symbol-package-id/main.bal

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) 2024, WSO2 LLC. (http://www.wso2.com).
+//
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied. See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+// Simple types
+type UserNil null;
+type UserBoolean boolean;
+type UserInt int;
+type UserFloat float;
+type UserDecimal decimal;
+
+// Sequence types
+type UserString string;
+type UserXml_1 xml;
+type UserXml_2 xml<xml:Element>;
+
+// Other types
+type UserAny any;
+type UserAnydata anydata;
+type UserNever never;
+type UserReadonly readonly;
+type UserJson json;
+type UserByte byte;
+type UserUnion_1 int|float|null;
```

Comment:
```suggestion
type UserUnion_1 int|float|();
```

Reply:
Fixed in 52976acc8b722e505c48db1b95c4370b271d5b7d.